packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -230,6 +232,25 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Extracts the orgId from the returned code from idbroker
+   * @instance
+   * @memberof AuthorizationBrowserFirstParty
+   * @param {String} code
+   * @private
+   * @returns {String}
+   */
+  _extractOrgIdFromCode(code) {
+    let orgId;
+
+    try {
+      orgId = code.split('_')[2];
+    } catch (error) {}
+    if (orgId) {
+      return orgId;
+    }
```

Comment:
Do we often use try/catch for value validation?

Reply:
I've changed it to not use a try/catch. Not sure it is any clearer though.